src/support/results/expectedResultsMobile.json

```diff
@@ -0,0 +1,236 @@
+    "viewport": [
+      "https://www.w3schools.com/w3images/wedding.jpg",
+      "/wp-content/rocket-test-data/images/file_example_JPG_100kB.jpg",
+      "/wp-content/rocket-test-data/images/lcp/testavif.avif",
+      "https://www.w3schools.com/w3images/rocks.jpg",
+      "/wp-content/rocket-test-data/images/lcp/testPng.png",
+      "/wp-content/rocket-test-data/images/maxime-lebrun-6g3Akg708E0-unsplash.jpg",
+      "https://www.w3schools.com/w3images/ocean.jpg"
+    ]
+  },
+  "lcp_attribute_template": {
+    "lcp": [
+      "/wp-content/rocket-test-data/images/wp-rocket.svg"
+    ],
+    "viewport": [
+      "https://new.rocketlabsqa.ovh/wp-content/rocket-test-data/images/lcp/testPng.png",
+      "https://new.rocketlabsqa.ovh/wp-content/rocket-test-data/images/wp-rocket2.svg",
+      "/wp-content/rocket-test-data/images/lcp/testsvg.svg",
+      "https://fastly.picsum.photos/id/976/200/300.jpg?hmac=s1Uz9fgJv32r8elfaIYn7pLpQXps7X9oYNwC5XirhO8",
+      "https://rocketlabsqa.ovh/wp-content/rocket-test-data/images/fixtheissue.jpg",
+      "/wp-content/rocket-test-data/images/lcp/testjpeg.jpeg",
+      "/wp-content/rocket-test-data/images/lcp/testavif.avif",
+      "/test.png"
+    ],
+    "enabled": true
+  },
+  "lcp_no_dimension_svg": {
+    "lcp": [
+      "/wp-content/rocket-test-data/images/lcp/testsvg.svg"
+    ],
+    "viewport": [],
+    "enabled": true
+  },
+  "lcp_no_dimensions_picture": {
+    "lcp": [
+      "https://e2e.rocketlabsqa.ovh/wp-content/rocket-test-data/images/lcp/testavif.avif"
+    ],
+    "viewport": [
+      "https://e2e.rocketlabsqa.ovh/wp-content/rocket-test-data/images/lcp/testjpg.jpg",
+      "/test.png"
+    ],
+    "enabled": true
+  },
+  "lcp_no_dimension_absolute_url": {
+    "lcp": [
+      "https://e2e.rocketlabsqa.ovh/wp-content/rocket-test-data/images/lcp/testjpg.jpg"
+    ],
+    "viewport": [
+      "https://e2e.rocketlabsqa.ovh/wp-content/rocket-test-data/images/lcp/testPng.png",
+      "https://cdn.pixabay.com/photo/2023/06/05/02/01/starry-sky-8041247_1280.jpg"
+    ],
+    "enabled": true
+  },
+  "lcp_image_withspecialchar_template": {
+    "lcp": [
+      "/wp-content/rocket-test-data/images/lcp/image%20with%20space.jpg"
+    ],
+    "viewport": [
+      "/wp-content/rocket-test-data/images/testspace%202.png",
+      "https://new.rocketlabsqa.ovh/wp-content/rocket-test-data/images/paperéquipesTest.jpeg"
+    ],
+    "enabled": false
+  },
+  "lcp_img_addedbydynamicstyle_template": {
+    "lcp": [
+      "/wp-content/rocket-test-data/images/paper.jpeg"
+    ],
+    "viewport": [
+      "https://new.rocketlabsqa.ovh/wp-content/rocket-test-data/images/lcp/testjpg.jpg"
+    ],
+    "enabled": true
+  },
+  "lcp_withfetchpriorityhigh_template": {
+    "lcp": [
+      "https://new.rocketlabsqa.ovh/wp-content/rocket-test-data/images/test_inline2.jpeg"
+    ],
+    "viewport": [
+      "/wp-content/rocket-test-data/images/lcp/testjpeg.jpeg"
+    ],
+    "enabled": true
+  },
+  "lcp_single_double": {
```

Comment:
What shall be the value here?

Reply:
if we don't handle this, then we can add the expected value and flag it as false so won't be executed for now. what do you think? https://wp-media.slack.com/archives/C070482U1TK/p1715763660411819